### ISSUE TITLE
Wildcard parameter name

### ISSUE THREAD

**Issue by antoooon** [husobee/vestigo#46]
When adding a wildcard parameter (i.e. *param instead of :param), the parameter name will be "_name" instead of the parameter name given in the pattern.

**Comment by husobee** [comment]
I guess the use case I was going for with the wildcard support was more like this:

```go 
    router := vestigo.NewRouter()
    router.Get("/static/*", http.StripPrefix("/static/", http.FileServer(http.Dir("/static")))
```
Where we have a "match all after the *" situation.  I am worried about confusion adding parameter names to the wildcard, because then we would have confusing situations like this:

```go
    router := vestigo.NewRouter()
    router.Get("/*wildcardparam1/foo", handler)
```
This becomes confusing for two reasons, should the wildcard act like a `:` param, or should the lookup for the matched portion of the route end up like `something/foo` or just `something` given the input url `/something/foo`?

I feel like wildcard parameters need only to be used to match to the end of URL, and if you need to access that wildcard parameter you can get it by means of `vestigo.Param(req, "_name")` or by performing a strip prefix handler.

If we could make named wildcard parameters less confusing, I would be up for adding support for it, but I feel like it will become confusing.

**Comment by antoooon** [comment]
Thanks for your answer.

Indeed, wildcards only makes sense at the end of a pattern.
But it's nice to fetch the matched string just like I do with other parameters. 
I had to read the source just to find out the magic parameter name "_name".

I guess "_name" is fine, as long as it's documented. I just stumbled over this when switching to vestigo from httprouter, which does save the wildcard parameter by the name given after "*".
